src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -373,6 +381,24 @@ def lvm_pv_type_label(device)
         format(_("PV of %s"), vg.basename)
       end
 
+      # Type label when the device is a snapshot LV
+      #
+      # @param device [Y2Storage::LvmLv]
+      # @return [String]
+      def snapshot_type_label(device)
+        snapshot_type = device.is?(:lvm_thin_snapshot) ? "Thin" : ""
+
+        # TRANSLATORS: %{snapshot_type} is the snapshot type, "Thin" or empty.
+        #              %{snapshot_origin} is the original volume name, e.g. "user-data".
+        label = format(
+          _("%{snapshot_type} Snapshot LV of %{snapshot_origin}"),
```

Comment:
For a non-thin snapshot this would leave a trailing space. Even more, this will not work very well for languages in which the adjective goes after the name. I would simply have two full strings for:

 - "Snapshot of %{origin}"
 - "Thin snapshot of %{origin}"

Breaking translatable string into pieces made of other translatable strings is not that great.